### PR TITLE
feat(core)!: Rename package to assertive-ts/core

### PR DESCRIPTION
**Pull request by JoseLion**
BREAKING CHANGE: Deprecates the `@stackbuilders/assertive-ts` package in favor of `assertive-ts/core` so we can group plugins under the assertive-ts namespace

### CODE REVIEW

**Review by ChristianSama — APPROVED** [review]
Hey Jose, looks good! So the idea is the create additional npm packages under @assertive-ts?

File: README.md [hunk 1]
```diff
@@ -29,39 +28,35 @@ A distinctive feature of AssertiveTS with other assertion libraries is that it l
...
 
-## Install
-
-```sh
-npm install --save-dev @stackbuilders/assertive-ts
-```
+## Packages
 
-Or:
+For conviniece, this library is split into package grouped within the same namespace:
```
Comment (well1791) [inline README.md:33]:
```suggestion
For convenience, this library is split into packages grouped within the same namespace:
```

File: README.md [hunk 2]
```diff
@@ -29,39 +28,35 @@ A distinctive feature of AssertiveTS with other assertion libraries is that it l
...
 
-```sh
-yarn add --dev @stackbuilders/assertive-ts
-```
+- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required to use the [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). This is package is the replacement for the deprecated `stackbuilders/assertive-ts` package.
```
Comment (well1791) [inline README.md:35]:
```suggestion
- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required for [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). Also, this package replaces the deprecated `stackbuilders/assertive-ts` package.
```

File: README.md [hunk 3]
```diff
@@ -29,39 +28,35 @@ A distinctive feature of AssertiveTS with other assertion libraries is that it l
...
 
 ## Usage
 
-Import the library in your test script:
+Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases to the function, such as `assert` and `assertThat`.
```
Comment (well1791) [inline README.md:39]:
```suggestion
Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases such as `assert` and `assertThat`.
```

**Review by well1791 — APPROVED** [review]
@JoseLion LGTM just a couple suggestion on wording

File: README.md (hunk 2 above)
Comment (well1791) [reply README.md:35]: This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.

File: README.md (hunk 3 above)
Comment (well1791) [reply README.md:39]: This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.

Comment (JoseLion) [reply README.md:39]: Makes sense, we have this part duplicate in the two readme files to provide better visibility of how the library works in GitHub as well as NPM 🙂

**Comment by JoseLion** [comment]
@ChristianSama yes exactly, with this change we can create additional packages (plugins) in this repo and publish them under the `@assertive-ts` namespace 🙂

**Pull request merged**